language/diem-framework/src/lib.rs

```diff
@@ -367,12 +368,25 @@ fn build_error_code_map(output_path: &str, sources: &[String], dep_path: &str) {
 
 pub fn generate_rust_transaction_builders() {
     let abis = transaction_builder_generator::read_abis(COMPILED_TRANSACTION_SCRIPTS_ABI_DIR)
-        .expect("Failed to read generated ABIs");
+        .expect("Failed to read generated ABIs")
+        .into_iter()
+        .filter_map(|abi| {
+            match abi {
+                // TODO remove this when updated to support script functions
+                ScriptABI::TransactionScript(abi) => Some(abi),
+                ScriptABI::ScriptFunction(_) => None,
+            }
+        })
+        .collect::<Vec<_>>();
     {
         let mut file = std::fs::File::create(TRANSACTION_BUILDERS_GENERATED_SOURCE_PATH)
             .expect("Failed to open file for Rust script build generation");
-        transaction_builder_generator::rust::output(&mut file, &abis, /* local types */ true)
-            .expect("Failed to generate Rust builders for Diem");
+        transaction_builder_generator::rust::output(
+            &mut file,
+            &abis[..],
```

Comment:
I don't think you need the `[..]` in `&abis[..]`

Reply:
I'll give it a shot :) Rust was complaining at me in a number of areas about not being able to convert a `Vec` to a slice, but I might have gotten a little overzealous in my slicing here.